src/app/pages/maps/search-map/search/search.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  Component, ElementRef, EventEmitter, NgZone, OnInit, Output,
+  ViewChild
+} from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { MapsAPILoader } from '@agm/core';
+import {} from 'googlemaps';
+import { Location } from "../entity/Location";
+
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss']
+})
+export class SearchComponent implements OnInit {
+
+  public searchControl: FormControl;
+
+  @Output()
+  newLocationEvent = new EventEmitter<Location>();
```

Comment:
no need to name it `event`, just `positionLocated`, `positionChanged` - meaning some action